File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        
        int i = 0;
        int n = nums.size();
        vector<int> ans(2*n,0);
        for(i=0;i<n;i++){
            ans[i] = nums[i];
        }
        int j;
        for(i=n,j=0;i<2*n,j<n;i++,j++){
            ans[i] = nums[j];
        }
        return ans;
    }
};

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        data = {}
        for index,value in enumerate(nums):
            data[value] = index
        ans = []
        for index,value in enumerate(nums):
            if (target-value) in data and data[target-value]!=index:
                return [index,data[target-value]]


        
        
        